python/pyspark/tests.py

```diff
@@ -1032,6 +1033,15 @@ def test_aggregator(self):
         from pyspark.sql import Aggregator as Agg
         # self.assertEqual((0, '100'), tuple(g.agg(Agg.first(df.key), Agg.last(df.value)).first()))
 
+    def test_help_command(self):
+        # Regression test for SPARK-5464
+        rdd = self.sc.parallelize(['{"foo":"bar"}', '{"foo":"baz"}'])
+        df = self.sqlCtx.jsonRDD(rdd)
+        # render_doc() reproduces the help() exception without printing output
+        pydoc.render_doc(df)
```

Comment:
This is the case that threw an exception.  The other two cases below are just to improve our test-coverage, since those classes also have custom `__getattr__` methods.